nomenclature/definitions/variable/technology/power-plant.yaml

```
@@ -655,8 +655,20 @@ Maximum Discharge|Electricity|Energy Storage System:
    unit: MW
 
 Maximum Storage|Electricity|Energy Storage System:
-   description: Maximum energy that can be stored by the energy storage system unit.
+   description: Maximum energy that can be stored by the energy storage systems.
    unit: GWh
+   
+Maximum Storage|Electricity|Energy Storage System|Lithium-Ion:
+   description: Maximum energy that can be stored by the Lithium Iron Batteries systems.
+   unit: GWh
+
+Maximum Storage|Electricity|Energy Storage System|Redox flow:
+   description: Maximum energy that can be stored by the Redox Flow Batteries systems.
+   unit: GWh
+
+Maximum Storage|Electricity|Energy Storage System|CAES:
```

avoid abbreviations where possible
```suggestion
Maximum Storage|Electricity|Energy Storage System|Compressed Air:
```